src/templates/apps/add-edit-skill.hbs

```diff
@@ -40,22 +40,22 @@
     <div class="form-group stacked">
         <label>Skill Properties</label>
         <label class="checkbox">
-            <input type="checkbox" name="value" {{checked skill.value}}/> Class Skill
+            <input type="checkbox" name="value" {{checked skill.value}}/> {{localize "SFRPG.SkillProficiencyLevelClassSkill"}}
         </label>
         <label class="checkbox">
-            <input type="checkbox" name="isTrainedOnly" {{checked skill.isTrainedOnly}}/> Trained Only
+            <input type="checkbox" name="isTrainedOnly" {{checked skill.isTrainedOnly}}/> {{localize "SFRPG.SkillTrainedOnly"}}
         </label>
         <label class="checkbox">
-            <input type="checkbox" name="hasArmorCheckPenalty" {{checked skill.hasArmorCheckPenalty}}/> Armor Check Penalty
+            <input type="checkbox" name="hasArmorCheckPenalty" {{checked skill.hasArmorCheckPenalty}}/> {{localize "SFRPG.ArmorCheckPenalty"}}
         </label>
         {{#if isNpc}}
         <label class="checkbox">
-            <input type="checkbox" name="enabled" {{checked skill.enabled}}/> Enabled
+            <input type="checkbox" name="enabled" {{checked skill.enabled}}/> {{localize "SFRPG.ModifierEnabledLabel"}}
         </label>
         {{/if}}
         {{#if canDelete}}
         <label class="checkbox">
-            <input type="checkbox" name="remove" /> Delete?
+            <input type="checkbox" name="remove" /> {{localize "SFRPG.Delete"}}?
```

Comment:
The question mark should be included in the localisation string. I'd also prefer if it was called `SFRPG.SkillDelete` or something.

Reply:
fixed